locksmith/src/websub/jobs/renewKeys.ts

```diff
@@ -0,0 +1,99 @@
+import { Op } from 'sequelize'
+import { networks } from '@unlock-protocol/networks'
+import { KeysToRenew } from '../../graphql/datasource'
+import { KeyRenewal } from '../../models'
+import { renewKey } from '../helpers'
+import { logger } from '../../logger'
+
+const FETCH_LIMIT = 25
+
+async function fetchKeysToRenew(network: number, page = 0) {
```

Comment:
Mind adding a few tests for this please?

Reply:
yep